proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -809,9 +735,7 @@ message GetValidatorParticipationRequest {
     }
 }
 
-// DEPRECATED: Prysm Web UI and associated endpoints will be fully removed in a future hard fork.
```

Comment:
also global participation rate should be looked at above

Reply:
the e2e can be fixed regarding it

Reply:
I don't understand this comment. We can't remove validator participation because it's not marked as deprecated.

Reply:
validator participating is not deprecated but there are fields in it that are

`float global_participation_rate = 1 [deprecated = true];` for example